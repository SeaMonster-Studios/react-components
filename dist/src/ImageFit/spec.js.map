;;AAAA;;;;AACA;;AACA;;IAAYA,O;;AACZ;;AACA;;;;;;AAEAC,UAAUC,4BAAV;;AAEAC,OAAOC,qBAAP,CAA6B,mCAAiBJ,OAAjB,CAA7B;;AAEAK,SAAS,UAAT,EAAqB,YAAM;AACzBC,KAAG,iFAAH,EAAsF,YAAM;AAAA,uBAC3DC,aAD2D;AAAA,QAClFC,WADkF,gBAClFA,WADkF;AAAA,QACrEC,KADqE,gBACrEA,KADqE;;AAE1F,QAAMC,WAAWF,YAAY,qBAAZ,CAAjB;;AAEA,QAAMG,iBAAiBC,OAAOC,gBAAP,CAAwBH,QAAxB,CAAvB;;AAEAP,WAAOQ,eAAeG,UAAtB,EAAkCC,IAAlC,mBACkBN,MAAMO,GADxB,2BACiDP,MAAMQ,QADvD;;AAIAd,WAAOO,QAAP,EAAiBQ,WAAjB;AACAf,WAAOO,QAAP,EAAiBS,eAAjB;AACD,GAZD;AAaD,CAdD;;AAgBA,SAASZ,WAAT,CAAqBa,SAArB,EAAgC;AAC9B,MAAMX;AACJY,SAAK,gCADD;AAEJC,SAAK,6CAFD;AAGJN,SAAK,OAHD;AAIJC,cAAU;;AAJN,KAMDG,SANC,CAAN;;AASA,MAAMG,UAAU,iCAAO,8BAAC,UAAD,EAAcd,KAAd,CAAP,CAAhB;;AAEA,2BACKc,OADL;AAEEd;AAFF;AAID