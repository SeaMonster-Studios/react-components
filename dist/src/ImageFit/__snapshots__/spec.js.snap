// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ImageFit Renders with the correct object styles applied from \`fit\` and \`position\` props. 1`] = `
.emotion-0 {
  font-family: 'object-fit: cover; object-position: top center';
  object-fit: cover;
  object-position: top center;
}

<img
  alt="something descriptive about my random image"
  class="emotion-0 emotion-1"
  data-ofi-src="https://picsum.photos/1200/500"
  data-ofi-srcset="null"
  data-testid="component-image-fit"
  src="data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='1' height='0'%3E%3C/svg%3E"
  srcset=""
  style="background-image: url(https://picsum.photos/1200/500); background-position: top center; background-repeat: no-repeat; background-origin: content-box; background-size: cover;"
/>
`;
